### PR TITLE
ADH-7533 add toggle chart info fullscreen

### PR DESCRIPTION
**Pull request by VasiliyTrue**
## Why are the changes needed?

Close #xxx.

## Brief change log

-

## How was this patch tested?

- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] Run test locally before making a pull request

## Documentation

- Does this pull request introduce a new feature? (yes / no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

File: amoro-web/vite.config.ts [hunk 1]
```diff
@@ -97,39 +97,39 @@ export default defineConfig({
...
+          options.headers = {
+            'Access-Control-Allow-Origin': '*',
+            'Access-Control-Allow-Credentials': 'true',
+            'Access-Control-Allow-Headers':
+                'Content-Type, Content-Length, Authorization, Accept, X-Requested-With , yourHeaderFeild',
+            'Access-Control-Allow-Methods': 'PUT,POST,GET,DELETE,OPTIONS'
+          }
+        }
+      }
+    }
```
Comment (asphix) [inline amoro-web/vite.config.ts]: not needed changes

**Comment by VasiliyTrue** [comment]
https://tracker.yandex.ru/ADH-7533

File: amoro-web/vite.config.ts (hunk 1 above)
Comment (VasiliyTrue) [reply amoro-web/vite.config.ts]: done
